src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Duke {
+    public static int MAX_TASK = 100;
+    static Task[] tasks = new Task[MAX_TASK];
+    public static int taskCount = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printHello();
+        handleCommand();
+        printGoodbye();
+    }
```

Comment:
I like that your main method consists of readable methods, making it easy for readers to understand their functionalities.